build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -236,6 +236,10 @@ service ActionCache {
 // used in subsequent calls (e.g. to
 // [Execute][build.bazel.remote.execution.v2.Execution.Execute]).
 //
+// Servers SHOULD support serving empty blobs even if they have not been
```

Comment:
I find this description somewhat problematic, in that it only conveys an idea/mindset, but doesn't describe any semantics:
- 'serving empty blobs' could be interpreted as only affecting ByteStream.Read() and ContentAddressableStorage.BatchReadBlobs(). But what about ContentAddressableStorage.FindMissingBlobs()? Would that always return absence or presence? That call doesn't 'serve' blobs in the literal sense.
- Is it valid to call ByteStream.Write() or ContentAddressableStorage.BatchUpdateBlobs() on empty blobs? If not, what kind of error should be returned?
- How does this affect ActionCache.GetActionResult()?
- Are we happy with the word 'SHOULD'? Shouldn't we use 'MUST' here?

Reply:
> 'serving empty blobs' could be interpreted as only affecting ByteStream.Read() and ContentAddressableStorage.BatchReadBlobs(). But what about ContentAddressableStorage.FindMissingBlobs()? Would that always return absence or presence? That call doesn't 'serve' blobs in the literal sense.

How about "Servers SHOULD behave as though empty blobs are always available, even if they have not been uploaded." ?

> Is it valid to call ByteStream.Write() or ContentAddressableStorage.BatchUpdateBlobs() on empty blobs? If not, what kind of error should be returned?

Yes, it is valid to explicitly upload empty blobs (as long as the hash matches, like with any other blob).

> How does this affect ActionCache.GetActionResult()?

As I understand it, this behaviour only refers to empty blobs in the CAS, so this comment is attached to the ContentAddressableStorage service.

> Are we happy with the word 'SHOULD'? Shouldn't we use 'MUST' here?

Right- I will fix that.

Reply:
> How about "Servers SHOULD behave as though empty blobs are always available, even if they have not been uploaded." ?

That sounds better. Thanks!